python/paddle/fluid/contrib/slim/tests/quantization/mkldnn_int8_quantization_strategy.py

```diff
@@ -0,0 +1,212 @@
+#   copyright (c) 2019 paddlepaddle authors. all rights reserved.
```

Comment:
Is this file a unit test? If so, it should be placed in the `python/paddle/fluid/contrib/slim/tests/` path. What's more, its name should be started with `test_`.

Reply:
`python/paddle/fluid/contrib/slim/tests/quantization` should only place some yaml config files, not python files.

Reply:
This is the suggestion comes form @wojtuss, he think that adding `test_mkldnn_int8_quantization_strategy.py` in `python/paddle/fluid/contrib/slim/tests/` and removed if not running on Linux platform compiled with MKLDNN is a bad idea.